khmer/_oxli/hllcounter.pxd

```diff
@@ -1,38 +1,47 @@
 from libcpp cimport bool
 from libcpp.memory cimport unique_ptr
-from libcpp.sting cimport string
+from libcpp.string cimport string
 from libcpp.vector cimport vector
 from libc.stdint cimport uint64_t
 
 from oxli_types cimport *
 from parsing cimport CpReadParser
 
+
+cdef extern from "oxli_exception_convert.hh":
+    cdef void oxli_raise_py_error()
+
+
 cdef extern from "oxli/hllcounter.hh" namespace "oxli":
     cdef cppclass CpHLLCounter "oxli::HLLCounter":
-        CpHLLCounter(double, WordLength)
-        CpHLLCounter(int, WordLength)
+        CpHLLCounter(double, WordLength) except +oxli_raise_py_error
+        CpHLLCounter(int, WordLength) except +oxli_raise_py_error
 
         void add(const string &)
         unsigned int consume_string(const string &)
         void consume_seqfile[SeqIO](const string &,
                                     bool,
                                     unsigned int &,
-                                    unsigned long long &)
+                                    uint64_t &) except +oxli_raise_py_error
 
-        void consume_seqfile[SeqIO](unique_ptr[CpReadParser[SeqIO]]&,
-                                    bool,
-                                    unsigned int &,
-                                    unsigned long long &)
+#        void consume_seqfile[SeqIO](unique_ptr[CpReadParser[SeqIO]]&,
+#                                    bool,
+#                                    unsigned int &,
+#                                    unsigned long long &)
```

Comment:
Would this https://github.com/dib-lab/khmer/pull/1675/files#diff-91092b4a16651bcd34d40d87061f3845R175 help with not having to comment this out?